resources/monitoring/charts/grafana/Chart.yaml

```diff
@@ -1,6 +1,7 @@
-apiVersion: v1
+apiVersion: v2
 name: grafana
-version: 6.1.9
+version: 6.11.0
+appVersion: 7.5.9
```

Comment:
We deleted the `appVersion` field from all of our charts intentionally, as people tend to forget maintaining it.